smart-contracts/package.json

```diff
@@ -48,6 +49,7 @@
     "ci": "npm run lint && npm run test",
     "build": "truffle compile --all",
     "zos": "zos",
+    "flatten": "truffle-flattener contracts/PublicLock.sol",
```

Comment:
Could you maybe make this write to a file instead, like add `> Flattened_PublicLock.sol` to the end of the command? When I flatten and output to the console, I can't scroll up enough to copy the whole thing...

Reply:
Sounds good.  Added.